### ISSUE TITLE
Favorites preset

### ISSUE THREAD

**Issue by jsupa** [jstrausd/homebridge-simple-wled#16]
How to add favourites preset to cfg? thanks

**Comment by jstrausd** [comment]
How do you mean "favorites preset"?
Currently it's only supported to add a own list of supported effects.

**Comment by etlam262** [comment]
I think they'd like to be able to select their own favorites as effects. I'd love to see that feature as well.

**Comment by faraww** [comment]
Me too. Currently choosing the own presets through mqtt, but I would be very nice, if it become possible with this plugin

**Comment by marcel293** [comment]
Yes, presets would be great!

**Comment by kopiro** [comment]
+1 for this (for context, this is the presets)

![image](https://user-images.githubusercontent.com/839700/207079280-a93894f1-19ef-4e3b-b411-904de97d244c.png)

**Comment by bite127** [comment]
Yes I need this feature too. The Presets featuere would be verry nice. Please add it to the plugin
